features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1120,6 +1120,22 @@
         <ResendConfirmationReCaptcha>{{identity_mgt.user_self_registration.enable_resend_confirmation_recaptcha}}</ResendConfirmationReCaptcha>
     </SelfRegistration>
 
+    <LiteRegistration>
+            <Enable>{{identity_mgt.lite_user_registration.enable}}</Enable>
+            <LockOnCreation>{{identity_mgt.lite_user_registration.lock_on_creation}}</LockOnCreation>
+            <Notification>
+                <InternallyManage>{{identity_mgt.lite_user_registration.notification.manage_internally}}</InternallyManage>
+            </Notification>
+            <ReCaptcha>{{identity_mgt.lite_user_registration.enable_recaptcha.enable}}</ReCaptcha>
```

Comment:
identity_mgt.lite_user_registration.enable_recaptcha.enable -> enable is repeated.
Change to identity_mgt.lite_user_registration.recaptcha.enable

Reply:
fixed in https://github.com/wso2/carbon-identity-framework/pull/3315/commits/531de2c815034e255c9dea99628c77c5d83c54a3